src/test/java/org/sagebionetworks/bridge/services/ParticipantFileServiceTest.java

```diff
@@ -153,6 +152,8 @@ public void createParticipantFile() {
         assertEquals(result.getUploadUrl(), upload);
         assertEquals(result.getAppId(), "api");
         assertEquals(TestConstants.TIMESTAMP.compareTo(result.getCreatedOn()), 0);
+        assertEquals(TestConstants.TIMESTAMP.plusDays(1).toDate(), result.getExpires().toDate());
```

Comment:
I'm not sure what the old code was doing, but you can just say

```
assertEquals(TestConstants.TIMESTAMP, result.getCreatedOn())
```

And similarly for expires.